File: javalin/src/test/java/io/javalin/jte/precompiled/JtetestGenerated.java
package io.javalin.jte.precompiled;

import io.javalin.TestTemplates.JteTestPage;

public final class JtetestGenerated {
    public static final String JTE_NAME = "test.jte";
    public static final int[] JTE_LINE_INFO = {0, 0, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 3};

    public static void render(gg.jte.html.HtmlTemplateOutput jteOutput, gg.jte.html.HtmlInterceptor jteHtmlInterceptor, JteTestPage page) {
        jteOutput.writeContent("<h1>");
        jteOutput.setContext("h1", null);
        jteOutput.writeUserContent(page.getHello());
        jteOutput.writeContent(" ");
        jteOutput.setContext("h1", null);
        jteOutput.writeUserContent(page.getWorld());
        jteOutput.writeContent("!</h1>\n");
    }

    public static void renderMap(gg.jte.html.HtmlTemplateOutput jteOutput, gg.jte.html.HtmlInterceptor jteHtmlInterceptor, java.util.Map<String, Object> params) {
        JteTestPage page = (JteTestPage) params.get("page");
        render(jteOutput, jteHtmlInterceptor, page);
    }
}

File: javalin/src/test/java/io/javalin/jte/precompiled/kte/JtetestGenerated.kt
package io.javalin.jte.precompiled.kte

import io.javalin.TestTemplates.JteTestPage

class JtetestGenerated {
    companion object {
        @JvmField
        val JTE_NAME = "kte/test.kte"
        @JvmField
        val JTE_LINE_INFO = intArrayOf(0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 3)
        @JvmStatic
        fun render(jteOutput: gg.jte.html.HtmlTemplateOutput, jteHtmlInterceptor: gg.jte.html.HtmlInterceptor?, page: JteTestPage) {
            jteOutput.writeContent("<h1>")
            jteOutput.setContext("h1", null)
            jteOutput.writeUserContent(page.hello)
            jteOutput.writeContent(" ")
            jteOutput.setContext("h1", null)
            jteOutput.writeUserContent(page.world)
            jteOutput.writeContent("!</h1>\n")
        }

        @JvmStatic
        fun renderMap(jteOutput: gg.jte.html.HtmlTemplateOutput, jteHtmlInterceptor: gg.jte.html.HtmlInterceptor?, params: Map<String, Any>) {
            val page = params.get("page") as JteTestPage
            render(jteOutput, jteHtmlInterceptor, page);
        }
    }
}
